addons/website_event_track_quiz/static/src/js/event_quiz.js

```diff
@@ -0,0 +1,706 @@
+                }
+            }
+            questionFormWidget.destroy();
+        },
+
+        /**
+         * After deletion of a Question the display is refreshed with the removal of the Question
+         * the reordering of all the remaining Questions and the change of the new Question sequence
+         * if the QuestionFormWidget is initialized.
+         *
+         * @param event
+         * @private
+         */
+        _deleteQuestion: function (event) {
+            var questionId = event.data.questionId;
+            this.$('.o_quiz_js_lesson_quiz_question[data-question-id=' + questionId + ']').remove();
+            this.quiz.questionsCount--;
+            this._reorderQuestions();
+            var $newQuestionSequence = this.$('.o_quiz_js_lesson_quiz_new_question .o_quiz_quiz_question_sequence');
+            $newQuestionSequence.text(parseInt($newQuestionSequence.text()) - 1);
+            if (this.quiz.questionsCount === 0 && !this.$('.o_wsildes_quiz_question_input').length) {
+                this.$('.o_quiz_js_quiz_add_quiz').removeClass('d-none');
+                this.$('.o_quiz_js_quiz_add_question').addClass('d-none');
+                this.$('.o_quiz_js_quiz_validation').addClass('d-none');
+            }
+        },
+    });
+
+    /**
+     * Dialog box shown when clicking the deletion button on a Question.
+     * When confirming it sends a RPC request to delete the Question.
+     */
+    var ConfirmationDialog = Dialog.extend({
+        template: 'quiz.confirm.deletion',
+        xmlDependencies: Dialog.prototype.xmlDependencies.concat(
+            ['/gamification_quiz/static/src/xml/quiz_create.xml']
+        ),
+
+        /**
+         * @override
+         * @param parent
+         * @param options
+         */
+        init: function (parent, options) {
+            options = _.defaults(options || {}, {
+                title: _t('Delete Question'),
+                buttons: [
+                    { text: _t('Yes'), classes: 'btn-primary', click: this._onConfirmClick },
+                    { text: _t('No'), close: true}
+                ],
+                size: 'medium'
+            });
+            this.questionId = options.questionId;
+            this.questionTitle = options.questionTitle;
+            this._super.apply(this, arguments);
+        },
+
+        /**
+         * Handler when the user confirm the deletion by clicking on 'Yes'
+         * it sends a RPC request to the server and triggers an event to
+         * visually delete the question.
+         * @private
+         */
+        _onConfirmClick: function () {
+            var self = this;
+            this._rpc({
+                model: 'quiz.question',
+                method: 'unlink',
+                args: [this.questionId],
+            }).then(function () {
+                self.trigger_up('delete_question', { questionId: self.questionId });
+                self.close();
+            });
+        }
+    });
```

Comment:
Same for all that stuff, used by frontend edition that is not available in our simplified version.